doc/experiment.rst

```diff
@@ -1,327 +1,423 @@
-    #SBATCH --time=00:10:00
+  If a launcher cannot be found or no `launcher` parameter is provided, the default value of
+  `launcher="local"` will be used.
 
-    python /path/to/script.py
-
-Ensemble
 ========
+Entities
+========
+Entities are SmartSim API objects that can be launched and
+managed on the compute system via the Experiment API. While the
+``Experiment`` object is intended to be instantiated once in a
+SmartSim driver script, there is no limit to the number of SmartSim entities
+within an Experiment. In the following subsections, we define the
+general purpose of the three entities that can be created via
+Experiment API factory methods:
+
+* ``Orchestrator``
+* ``Model``
+* ``Ensemble``
+
+To create a reference to each entity object, use the associated
+``Experiment.create_...()`` function.
+
+.. list-table:: Experiment API Entity Creation
+   :widths: 20 65 25
+   :header-rows: 1
+
+   * - Factory Method
+     - Example
+     - Return Type
+   * - ``create_database()``
+     - ``orch = exp.create_database([port, db_nodes, ...])``
+     - :ref:`Orchestrator <orc_api>`
+   * - ``create_model()``
+     - ``model = exp.create_model(name, run_settings)``
+     - :ref:`Model <mode_api>`
+   * - ``create_ensemble()``
+     - ``ensemble = exp.create_ensemble(name[, params, ...])``
+     - :ref:`Ensemble <ensem_api>`
+
+Each entity instance can be used to start,
+monitor, and stop simulations from the notebook
+using the :ref:`Experiment API<experiment_api>`.
+
+.. list-table:: Interact with Entities during the Experiment
+   :widths: 25 55 25
+   :header-rows: 1
+
+   * - Experiment Function
+     - Example
+     - Desc
+   * - ``start()``
+     - ``exp.start(*args[, block, summary, ...])``
+     - Launch an Entity
+   * - ``stop()``
+     - ``exp.stop(*args)``
+     - Clobber an Entity
+   * - ``get_status()``
+     - ``exp.get_status(*args)``
+     - Retrieve Entity Status
+
+Orchestrator
+------------
+The orchestrator is an in-memory database with features designed
+to enable a wide variety of AI-enabled workflows, including features
+for online training, low-latency inference, cross-application data
+exchange, online interactive visualization, online data analysis, computational
+steering, and more. The ``Orchestrator`` can be thought of as a general
+feature store capable of storing numerical data, ML models, and scripts
+and capable of performing inference and script evaluation on feature
+store data. Any SmartSim ``Model`` or ``Ensemble`` model can connect to the
+``Orchestrator`` via the :ref:`SmartRedis<SmartRedis Client Library Hook>`
+client library to transmit data, execute ML models, and execute scripts.
+
+**SmartSim offers two types Orchestrator deployments:**
+
+* :ref:`Clustered Orchestrator <Clustered Orchestrator>`
+* :ref:`Colocated Orchestrator <Colocated Orchestrator>`
+
+Clustered Orchestrator
+^^^^^^^^^^^^^^^^^^^^^^
+The orchestrator can be composed of one or more in-memory database shards that are spread
```

Comment:
Simplify this sentence to "The standard orchestrator can be deployed on a single compute node or can be sharded (distributed) over multiple nodes."